src/acceptance-test/java/org/zalando/nakadi/webservice/hila/SubscriptionAT.java

```diff
@@ -390,6 +392,52 @@ public void whenStatsOnNotInitializedSubscriptionThanCorrectResponse() throws IO
         }
     }
 
+    @Test
+    public void whenLightStatsOnNotInitializedSubscriptionThenCorrectResponse() throws IOException {
+        final String et = createEventType().getName();
+        final Subscription s = createSubscriptionForEventType(et);
+        final Response response = when().get("/subscriptions?show_status=true").thenReturn();
+        final ItemsWrapper<Subscription> subsItems = MAPPER.readValue(response.print(),
+                new TypeReference<ItemsWrapper<Subscription>>(){});
+        for (final Subscription subscription: subsItems.getItems()) {
+            if (subscription.getId().equals(s.getId())) {
+                Assert.assertNotNull(subscription.getStats());
+                Assert.assertEquals("unassigned", subscription.getStats().get(0).getPartitions().get(0).getState());
+                Assert.assertEquals("", subscription.getStats().get(0).getPartitions().get(0).getStreamId());
+                return;
+            }
+        }
+        Assert.assertTrue(false);
+    }
+
+    @Test
+    public void whenLightStatsOnActiveSubscriptionThenCorrectRespones() throws IOException {
+        final String et = createEventType().getName();
+        final Subscription s = createSubscriptionForEventTypeFromBegin(et);
+
+        publishEvents(et, 15, i -> "{\"foo\":\"bar\"}");
+
+        final TestStreamingClient client = TestStreamingClient
+                .create(URL, s.getId(), "max_uncommitted_events=20")
+                .start();
+        waitFor(() -> assertThat(client.getBatches(), hasSize(15)));
+
+        final Response response = when().get("/subscriptions?show_status=true").thenReturn();
+        final ItemsWrapper<Subscription> subsItems = MAPPER.readValue(response.print(),
+                new TypeReference<ItemsWrapper<Subscription>>(){});
+        for (final Subscription subscription: subsItems.getItems()) {
+            if (subscription.getId().equals(s.getId())) {
+                Assert.assertNotNull(subscription.getStats());
+                Assert.assertEquals("assigned", subscription.getStats().get(0).getPartitions().get(0).getState());
+                Assert.assertNotEquals("", subscription.getStats().get(0).getPartitions().get(0).getStreamId());
```

Comment:
you can actually take streamId from TestStreamingClient and use it here for assertion.